pkg/login/social/social.go

```diff
@@ -65,36 +65,31 @@ func NewOAuthService() {
 	for _, name := range allOauthes {
 		sec := setting.Raw.Section("auth." + name)
 		info := &setting.OAuthInfo{
-			ClientId:                     sec.Key("client_id").String(),
-			ClientSecret:                 sec.Key("client_secret").String(),
-			Scopes:                       util.SplitString(sec.Key("scopes").String()),
-			AuthUrl:                      sec.Key("auth_url").String(),
-			TokenUrl:                     sec.Key("token_url").String(),
-			ApiUrl:                       sec.Key("api_url").String(),
-			Enabled:                      sec.Key("enabled").MustBool(),
-			EmailAttributeName:           sec.Key("email_attribute_name").String(),
-			EmailAttributePath:           sec.Key("email_attribute_path").String(),
-			AllowedDomains:               util.SplitString(sec.Key("allowed_domains").String()),
-			HostedDomain:                 sec.Key("hosted_domain").String(),
-			AllowSignup:                  sec.Key("allow_sign_up").MustBool(),
-			Name:                         sec.Key("name").MustString(name),
-			TlsClientCert:                sec.Key("tls_client_cert").String(),
-			TlsClientKey:                 sec.Key("tls_client_key").String(),
-			TlsClientCa:                  sec.Key("tls_client_ca").String(),
-			TlsSkipVerify:                sec.Key("tls_skip_verify_insecure").MustBool(),
+			ClientId:           sec.Key("client_id").String(),
+			ClientSecret:       sec.Key("client_secret").String(),
+			Scopes:             util.SplitString(sec.Key("scopes").String()),
+			AuthUrl:            sec.Key("auth_url").String(),
+			TokenUrl:           sec.Key("token_url").String(),
+			ApiUrl:             sec.Key("api_url").String(),
+			Enabled:            sec.Key("enabled").MustBool(),
+			EmailAttributeName: sec.Key("email_attribute_name").String(),
+			EmailAttributePath: sec.Key("email_attribute_path").String(),
+			AllowedDomains:     util.SplitString(sec.Key("allowed_domains").String()),
+			HostedDomain:       sec.Key("hosted_domain").String(),
+			AllowSignup:        sec.Key("allow_sign_up").MustBool(),
+			Name:               sec.Key("name").MustString(name),
+			TlsClientCert:      sec.Key("tls_client_cert").String(),
+			TlsClientKey:       sec.Key("tls_client_key").String(),
+			TlsClientCa:        sec.Key("tls_client_ca").String(),
+			TlsSkipVerify:      sec.Key("tls_skip_verify_insecure").MustBool(),
+			// TODO: Deprecate send_client_credentials_via_post as auth style is autodetected
```

Comment:
what was changed?

Reply:
I added comment and gofmt reformated structure initialization